.github/workflows/build.yml

```
@@ -0,0 +1,175 @@
+          if [ -z "$DF" ] || [ "$DF" = "null" ]; then
+            echo "No dockerfile entry found for \"$SERVICE\" in build-config.yml; using default"
+            DF="$DEFAULT_DOCKERFILE"
+          else
+            echo "Found dockerfile \"$DF\" for service \"$SERVICE\"."
+          fi
+
+          echo "dockerfile_path=$DF" >> "$GITHUB_OUTPUT"
+          echo "DOCKERFILE_PATH=$DF" >> "$GITHUB_ENV"
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.7)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)